.github/workflows/publish-hotfix.yml

```diff
@@ -0,0 +1,182 @@
+# Description: this workflow publishes UI packages to `latest` tag on NPM.
+#
+# Triggered by: whenever a PR is merged to `hotfix` branch
+
+# If successful, we will publish all updated UI packages to
+#   - https://www.npmjs.com/package/@aws-amplify/ui
+#   - https://www.npmjs.com/package/@aws-amplify/ui-react
+#   - https://www.npmjs.com/package/@aws-amplify/ui-react-native
+#   - https://www.npmjs.com/package/@aws-amplify/ui-vue
+#   - https://www.npmjs.com/package/@aws-amplify/ui-angular
+
+name: Test and Publish / hotfix
+
+on:
+  push:
+    branches: [hotfix]
+
+permissions:
+  id-token: write # This is required for requesting the JWT
+  contents: write # Used to push tags to GitHub
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      has-changesets: ${{ steps.has-changesets.outputs.has-changesets }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f # v3.4.0 https://github.com/actions/checkout/commit/24cb9080177205b6e8c946b17badbe402adc938f
+        with:
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
+      - name: Check if repo has unconsumed changesets
+        id: has-changesets
+        uses: ./.github/actions/has-changesets
+      # Note: if there are no changesets, that means that either
+      #         (1) A hot fix PR PR was just merged, or
+      #         (2) no commits with changesets have been merged after packages were last published.
```

Comment:
Wonder if case (2) is really relevant here because maintainers are instructed to add and consume changesets in one PR. That said, maybe it's just nice to leave it here if we do make a mistake?

Reply:
Yeah, was thinking it would be nice to have in case of mistakes